spec/controllers/nps_surveys/api_internal/responses_controller_spec.rb

```diff
@@ -2,30 +2,43 @@
 
 module NpsSurveys
   describe ApiInternal::ResponsesController do
-    include_context 'oauth authentication'
+    before(:each) do
+      user = FactoryGirl.create(:user)
```

Comment:
It is not a variable declaration, I need to overwrite current user, can you do that with a let?